build/Packages.props

```diff
@@ -59,6 +59,12 @@
         <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.14.114" />
         <PackageReference Update="Microsoft.VisualStudio.SDK.EmbedInteropTypes" Version="15.0.12" />
 
+        <!-- The two references below can be deleted once OmniSharp.Extensions.LanguageServer
```

Comment:
O#-roslyn and Razor both indirectly reference `Microsoft.VisualStudio.Validation`:

- O#-roslyn via `OmniSharp.Extensions.LanguageServer` -> `OmniSharp.Extensions.JsonRpc` -> `Nerdbank.Streams` -> `Microsoft.VisualStudio.Threading` -> `Microsoft.VisualStudio.Validation`
- Razor via `Microsoft.VisualStudio.Threading` -> `Microsoft.VisualStudio.Validation`

Razor is on a 17.xx version of `Microsoft.VisualStudio.Validation`, while O#-roslyn is on a lower version. We need to bump O#-roslyn's version to avoid errors in Razor files. However, the newest version of `OmniSharp.Extensions.LanguageServer` only utilizes a Validation version of 16.xx, so we have to bump explicitly until they update to a newer version.